src/main/scala/io/iohk/ethereum/App.scala

```diff
@@ -0,0 +1,32 @@
+package io.iohk.ethereum
+
+import java.net.{InetSocketAddress, URI}
+
+import akka.actor.ActorSystem
+import io.iohk.ethereum.crypto._
+import io.iohk.ethereum.network.{NodeInfo, ServerActor, PeerManagerActor}
+
+object App {
+
+  val nodeKey = generateKeyPair()
+
+  def main(args: Array[String]): Unit = {
+    val listenHostname = "127.0.0.1"
+    val listenPort = 9076
```

Comment:
Feel free to ignore this comment but I would read these values from config file.  The sooner we create it the less time we'll have to spend on grepping code for all possible config values.